examples/support/analysis/README.md

```diff
@@ -0,0 +1,237 @@
+      keyObjective: true
+  totalScore:
+    passPercentage: 60
+    warningPercentage: 50
+EOF
+```
+
+## Executing an Analysis
+
+Finally, now that we have all of our metrics and our goals defined, we can apply an instance of the `Analysis` CRD
+to perform an analysis for a specific timeframe:
+
+```shell
+cat <<EOF | kubectl apply -f - 
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: Analysis
+metadata:
+  name: analysis-sample
+  namespace: analysis-demo
+spec:
+  timeframe:
+    from: 2023-09-25T13:10:00Z
+    to: 2023-09-25T13:15:00Z
+  args:
+    "workload": "podtato-head-frontend"
+  analysisDefinition:
+    name: my-analysis-definition
+EOF
+```
+
+Once applied, the status of the analysis can be checked with:
+
+```shell
+kubectl get analyses -n analysis-demo
+```
+
+This command should yield a list of all analyses within our `analysis-demo` namespace,
+together with the current status of the analysis.
+In our case, we will receive one analysis which has already completed and has passed:
+
+Please note that this example uses real data from an actual service monitored by Prometheus,
+so it could very well be that the result of your analysis might be a different one,
+as those values heavily depend on the environment this example is executed in.
+
+```shell
+NAME              ANALYSISDEFINITION       STATE       WARNING   PASS
+analysis-sample   my-analysis-definition   Completed             true
+```
+
+To get further details on the analysis, we can also retrieve the complete yaml representation:
+
+```shell
+kubectl get analyses -n analysis-demo analysis-sample -oyaml 
+```
+
+This will return something like the following, and includes the `status.raw` field, which contains a detailed
+breakdown on the result of each objective evaluation.
+
+```yaml
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: Analysis
+metadata:
+  name: analysis-sample
+  namespace: analysis-demo
+spec:
+  analysisDefinition:
+    name: my-analysis-definition
+  args:
+    workload: podtato-head-frontend
+  timeframe:
+    from: "2023-09-25T13:10:00Z"
+    to: "2023-09-25T13:15:00Z"
+status:
+  pass: true
+  raw: '{"objectiveResults":[{"result":{"failResult":{"operator":{"greaterThan":{"fixedValue":"500m"}},"fulfilled":false},"warnResult":{"operator":{"greaterThan":{"fixedValue":"300m"}},"fulfilled":false},"warning":false,"pass":true},"objective":{"analysisValueTemplateRef":{"name":"response-time-p95"},"target":{"failure":{"greaterThan":{"fixedValue":"500m"}},"warning":{"greaterThan":{"fixedValue":"300m"}}},"weight":1},"value":0.00475,"score":1},{"result":{"failResult":{"operator":{"greaterThan":{"fixedValue":"0"}},"fulfilled":false},"warnResult":{"operator":{},"fulfilled":false},"warning":false,"pass":true},"objective":{"analysisValueTemplateRef":{"name":"error-rate"},"target":{"failure":{"greaterThan":{"fixedValue":"0"}}},"weight":1,"keyObjective":true},"value":0,"score":1}],"totalScore":2,"maximumScore":2,"pass":true,"warning":false}'
+  state: Completed
+```
```

Comment:
Should we add a reflection on the result? Ex. The analysis's overall result is pass. It has been computed starting from each objective score. Each of the two objective have score  was1 if they were satisfied, with a weight of 1 and we thus got an overall score of 2. We got thus 100% and the analysis pass criteria was 60% so we pass.

Reply:
yes, will add a section explaining that